angular.json

```diff
@@ -35,13 +35,7 @@
           },
           "configurations": {
             "production": {
-              "budgets": [
-                {
-                  "type": "anyComponentStyle",
-                  "maximumWarning": "2kb",
-                  "maximumError": "4kb"
-                }
-              ],
+              "budgets": [],
```

Comment:
It seems `anyComponentStyle` `4kb` is too strict for the examples scss files. I think it doesn't matter for the examples so I removed it. Another option is increasing the size to around `30kb`. @beeman What do you think?

Reply:
> It seems `anyComponentStyle` `4kb` is too strict for the examples scss files. I think it doesn't matter for the examples so I removed it. Another option is increasing the size to around `30kb`. @beeman What do you think?

Happy to remove the restriction considering it's for the example app.